indra/newview/llpanelface.cpp

```diff
@@ -1325,14 +1324,6 @@ void LLPanelFace::updateUI(bool force_set_values /*false*/)
             }
         }
 
-        // Alpha gamma.  This should always be enabled, no matter if the object is mod or not.
-        {
-            U8   alpha_gamma           = 100;
-            bool identical_alpha_gamma = false;
-            LLSelectedTE::getAlphaGamma(alpha_gamma, identical_alpha_gamma);
-            mComboAlphaGamma->getSelectionInterface()->selectByValue(alpha_gamma);
-            mComboAlphaGamma->setEnabled(true);
-        }
```

Comment:
This moved down below so the UI could enabled even when permissions are **no-mod**.